docs/src/prototypes/IconViewer/index.tsx

```diff
@@ -21,48 +50,67 @@ class IconViewerExample extends React.Component<any, {}> {
             styles: { fontSize: '16px' },
           }}
         />
-        <Provider theme={themeWithProcessedIcons}>
-          <Provider.Consumer
-            render={theme => (
-              <div>
-                <div>
-                  <Divider>
-                    <Header as="h3" content="Regular" />
-                  </Divider>
-                  <Grid columns={4} style={{ textAlign: 'center' }}>
-                    {Object.keys(theme.icons)
-                      .filter(name => name.startsWith(processedIconsNamePrefix))
-                      .sort()
-                      .map(name => (
-                        <div key={name} style={cellStyles}>
-                          <Icon name={name} />
-                          <br />
-                          <code>{name.replace(processedIconsNamePrefix, '')}</code>
-                        </div>
-                      ))}
-                  </Grid>
-                </div>
+
+        <div>
+          <Menu pointing tabular>
+            {Object.keys(this.iconFilters).map(filterName => (
+              <Menu.Item
+                name={filterName}
+                active={this.state.filter === filterName}
+                onClick={() => this.setState({ filter: filterName })}
+              />
+            ))}
+          </Menu>
+
+          <Provider theme={themeWithProcessedIcons}>
+            <Provider.Consumer
+              render={theme => (
                 <div>
-                  <Divider>
-                    <Header as="h3" content="Outline" />
-                  </Divider>
-                  <Grid columns={4} style={{ textAlign: 'center' }}>
-                    {Object.keys(theme.icons)
-                      .filter(name => name.startsWith(processedIconsNamePrefix))
-                      .sort()
-                      .map(name => (
-                        <div key={`${name}-outline`} style={cellStyles}>
-                          <Icon name={name} variables={{ outline: true }} />
-                          <br />
-                          <code>{name.replace(processedIconsNamePrefix, '')} outline</code>
-                        </div>
-                      ))}
-                  </Grid>
+                  <div>
+                    <Divider>
+                      <Header as="h3" content="Regular" />
+                    </Divider>
```

Comment:
For simplicity's sake, probably don't need the Divider wrapping the Header here.  The Header alone will cause a line break.

***

Side note, we are missing a "dividing" Header which would accomplish a similar look:
https://react.semantic-ui.com/elements/header/#variations-dividing

![image](https://user-images.githubusercontent.com/5067638/49664087-af710400-fa05-11e8-8f20-745c53f9edd6.png)